### ISSUE TITLE
Speech feedback

### ISSUE THREAD

**Issue by jpmckinney** [popolo-project/popolo-spec#74]
## Discussion

The [inventory of terms](http://www.popoloproject.com/appendices/terms.html#Speech) found terms for:
- section/group: it's possible that sections (groups of speeches) can be organized as Events (#22)
- keywords/tags: e.g. speeches can be tagged with issue areas
- language in which the speech is spoken (versus the language in which it is transcribed, which in RDF would be encoded with a language tag)
- end time: this doesn't seem to be commonly used

Who is using such terms? What are their use cases? Should the terms be added to the spec?
## Changes
- Added [`video`](http://schema.org/video) property

**Comment by akuckartz** [comment]
I will start with the last one: `language`. In JSON-LD all strings with human readable  text should be annotated with a language. (This does not necessarily answer the question.)

See: http://www.popoloproject.com/appendices/multilingualization.html and http://www.w3.org/TR/json-ld/#string-internationalization

**Comment by jpmckinney** [comment]
Indeed. With transcripts of spoken words, it can be relevant to note the original language in which the words were spoken, especially if the data is made available in multiple languages. Canada's House of Commons does this, for example. This is in contrast to multilingual documents or laws, where either language version is as official as the other, like in Canada, and neither has special status over the other. Spoken words in a multilingual discussion are a different situation; one version is the original and the other is a translation.

**Comment by girogiro** [comment]
What about renaming of `type` property to `classification` to be consistent with motion, organization and area?

**Comment by jpmckinney** [comment]
ContactDetail also has a `type`. In RDF, the ContactDetail and Speech `type` actually controls the class. This is good in RDF, because a `Narrative`, `Scene`, `Summary` or `Other` from Akoma Ntoso does not have a speaker, whereas a `Speech`, `Question` or `Answer` from Akoma Ntoso does. Since they have different properties, they should be different classes. In the JSON, they are merged into one for simplicity of implementation, but using `type` lets us keep a clean data model. On Motion, Organization, and Area, classification is a true classification, and is not a way of identifying the class.

**Comment by girogiro** [comment]
Slovak parliament provides a link to a videorecording for each speech instead of audio. I guess it will be quite common practice in other legislatures too. A new property `video` would be useful. Or, properties for audio and video links might be substituted by a `links` property.

**Comment by jpmckinney** [comment]
To go in a `links` property, we would need to add another property to Link for storing the code for the link (e.g. `audio`, `video`). We can't use `note` for storing the code, because `note` generally controls the link text. Since I don't anticipate there being a growing list of specific links, we can add a `video` property from http://schema.org/video

**Comment by akuckartz** [comment]
I suggest to continue the discussion on video and audio in #75. Media files or streams are a different matter than language or keywords.
